gnovm/stdlibs/std/banker.go

```diff
@@ -3,160 +3,89 @@ package std
 import (
 	"fmt"
 
+	gno "github.com/gnolang/gno/gnovm/pkg/gnolang"
 	"github.com/gnolang/gno/tm2/pkg/crypto"
 	"github.com/gnolang/gno/tm2/pkg/std"
 )
 
-// This has the same interface as stdlibs/std.Banker.
-// The native implementation of Banker (wrapped by any
-// wrappers for limiting functionality as necessary)
-// becomes available in Gno that implements
-// stdlibs/std.Banker.
-type Banker interface {
+// BankerInterface is the interface through which Gno is capable of accessing
+// the blockchain's banker.
+//
+// The name is what it is to avoid a collision with Gno's Banker, when
+// transpiling.
+type BankerInterface interface {
```

Comment:
Transpiled code has other conflicts, for instance between `native.go` and `native.gno.gen.go`:

```
$ go build -tags gno gnovm/stdlibs/std
# github.com/gnolang/gno/gnovm/stdlibs/std
./native.go:10:6: AssertOriginCall redeclared in this block
	native.gno:3:6[/home/tom/src/gno/gnovm/stdlibs/std/native.gno.gen.go:8:6]: other declaration of AssertOriginCall
```

 I know this was already there before your change, but I need to understand why you need to fix that one for `Banker`, but not the other ones like `AssertOriginCall`.

Reply:
@tbruyelle yes, of course, I haven't clearly explained that!

This PR was split from #1695. I've not done a perfect job with a split, this change would have better belonged there, but in that PR we're resolving the conflict for functions like `AssertOriginCall` by removing the functions without a definition. [This is what I mean](https://github.com/gnolang/gno/blob/9702dcb01d649d802b37e022df51dc76e4b96279/gnovm/pkg/transpiler/transpiler.go#L305-L325)

So, there is a problem for identifiers which exist in go and gno; but not for those that are linked together through native bindings, which are handled correctly.

Reply:
Got it thank you!